<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.selenium.testcases.portfoliosuite.PortFolioTest" tests="2" time="154.911" ignored="0" failures="0" timestamp="06 May 2019 12:59:38 GMT" skipped="0" hostname="Manish-PC" errors="1">
  <testcase name="createPortFolioTest" time="80.323" classname="com.selenium.testcases.portfoliosuite.PortFolioTest"/>
  <testcase name="deletePortFolioTest" time="74.588" classname="com.selenium.testcases.portfoliosuite.PortFolioTest">
    <error type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.selenium.driver.DriverScript.executeKeywords(DriverScript.java:48)
	at com.selenium.testcases.portfoliosuite.PortFolioTest.deletePortFolioTest(PortFolioTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: The following asserts failed:
	Could not delete the option
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.selenium.keywords.GenericKeywords.assertAll(GenericKeywords.java:384)
	at com.selenium.keywords.GenericKeywords.reportFailure(GenericKeywords.java:379)
	at com.selenium.keywords.GenericKeywords.validateElementNotInList(GenericKeywords.java:249)
	... 31 more
]]>
    </error>
  </testcase> <!-- deletePortFolioTest -->
</testsuite> <!-- com.selenium.testcases.portfoliosuite.PortFolioTest -->
